WordPress/src/main/res/values/strings.xml

```diff
@@ -263,6 +263,7 @@
     <!-- post view -->
     <string name="post_id">Post</string>
     <string name="upload_failed">Upload failed</string>
+    <string name="draft_uploaded">Draft uploaded</string>
```

Comment:
I added [a note](https://github.com/wordpress-mobile/WordPress-Android/projects/6#card-3588591) to the "Improve posting experience" column of the Media project - let's get a broader language design pass as part of the async overhaul.